Source/WebKit/WebProcess/WebPage/WebPage.messages.in

```diff
@@ -47,6 +47,7 @@ messages -> WebPage LegacyReceiver {
     ExecuteEditCommandWithCallback(String name, String argument) -> ()
     KeyEvent(WebKit::WebKeyboardEvent event)
     MouseEvent(WebKit::WebMouseEvent event, std::optional<Vector<WebKit::SandboxExtension::Handle>> sandboxExtensions)
+    GenerateSyntheticCommandP() -> (bool success)
```

Comment:
Nit - it might also be good to rename `success` here to either `handled` or `prevented`, for clarity?